src/svg/ds6/icons.svg

```diff
@@ -610,4 +610,52 @@
     <symbol  id="icon-eek-arrow" viewBox="0 0 11 28">
         <path d="M0 27.75H1.13239C1.76009 27.75 2.35161 27.4145 2.73099 26.8434L10.342 15.3851C10.886 14.5662 10.886 13.4338 10.342 12.6149L2.73099 1.15661C2.35161 0.585476 1.76009 0.25 1.13239 0.25H3.09944e-06" stroke="black" stroke-width="1"/>
     </symbol>
+    <symbol id="icon-bag"  viewBox="0 0 24 24">
+        <path fill="#111820" d="M17.9959 6.775C17.8776 3.5657 15.2384 1 12 1 8.6863 1 6 3.6863 6 7v2H4a1 1 0 00-.9864.8356l-2 12A1 1 0 002 23h20a.9998.9998 0 00.9864-1.1644l-2-12A1 1 0 0020 9h-2V7l-.0041-.225zM16 11v2l.0067.1166c.0578.4974.4805.8834.9933.8834.5523 0 1-.4477 1-1v-2h1.1529l1.6666 10H3.1805L4.847 11H6v2l.0067.1166C6.0645 13.614 6.4872 14 7 14c.5523 0 1-.4477 1-1v-2h8zm0-2H8V7l.0049-.1996C8.1089 4.684 9.8578 3 12 3c2.2091 0 4 1.7909 4 4v2z"/>
```

Comment:
We should look into whether all of these fills are necessary. I don't think they are.